gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/schema.yml

```diff
@@ -14,6 +14,7 @@ databaseChangeLog:
               - column: {name: description, type: nvarchar(256), constraints: { nullable: true } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.